src/audio/src/src.c

```diff
@@ -824,32 +821,6 @@ static int src_prepare(struct comp_dev *dev)
 	if (ret == COMP_STATUS_STATE_ALREADY_SET)
 		return PPL_STATUS_PATH_STOP;
 
-	/* SRC supports S16_LE, S24_4LE and S32_LE formats */
-	switch (dev->params.frame_fmt) {
-	case SOF_IPC_FRAME_S16_LE:
-		cd->data_shift = 0;
-		cd->polyphase_func = src_polyphase_stage_cir_s16;
-		/* Copy function is set by default in params() for 32 bit
-		 * data. Change it to 16 bit version here if source and sink
-		 * rates are equal.
-		 */
-		if (cd->source_rate == cd->sink_rate)
-			cd->src_func = src_copy_s16;
-		break;
-	case SOF_IPC_FRAME_S24_4LE:
-		cd->data_shift = 8;
-		cd->polyphase_func = src_polyphase_stage_cir;
-		break;
-	case SOF_IPC_FRAME_S32_LE:
-		cd->data_shift = 0;
-		cd->polyphase_func = src_polyphase_stage_cir;
-		break;
-	default:
-		trace_src_error_with_ids(dev, "src_prepare() error: "
-					 "invalid dev->frame_fmt");
-		return -EINVAL;
-	}
-
```

Comment:
I think I'm a little bit lost. Why are you removing this code?

Reply:
There was a conflict with master. I resolve it by removing this lines, same as in master. Commits `f4084551` and `703e8269`